SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -90,6 +93,7 @@ CNEMOEulerSolver::CNEMOEulerSolver(CGeometry *geometry, CConfig *config,
 
   }
 
+  //TODO: we dont need this in NEMO delete me
```

Comment:
go ahead then :P

Reply:
Hahah I wasnt sure, if we had any use, but Ill go ahead and delete

Reply:
Done